python/lib/sift_py/rest.py

```diff
@@ -14,3 +15,16 @@ class SiftRestConfig(TypedDict):
     uri: str
     apikey: str
     use_ssl: NotRequired[bool]
+
+
+def compute_uri(restconf: SiftRestConfig) -> str:
+    uri = restconf["uri"]
+
+    scheme_match = re.match(r"^(\w+://).+", uri)
+    if scheme_match:
+        raise Exception(f"The URL scheme '{scheme_match.groups()[0]}' should not be included")
```

Comment:
There is a `_clean_uri` util from `lib/sift_py/grpc/transport.py` that we extract and put into a top level internal util so folks can include `http://` or `https://` if they want. Folks always miss this detail and they get really funky error messages that isn't immediately obvious for anyone what went wrong.